src/builder/modules/Generator.res

```diff
@@ -122,8 +122,37 @@ let usePage = id => {
   model := Builder.usePage(model.contents, id)
 }
 
-let fillBackgroundColor = (fillStyle: string) => {
-  model := Generator_Builder.fillBackgroundColor(model.contents, fillStyle)
+let fillBackgroundColor = (color: string) => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, color)
+}
+
+let fillBackgroundColorWithWhite = () => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, "#ffffff")
+}
+
+let drawLine = (
+  from: Builder.position,
+  to: Builder.position,
+  ~color: string="#000000",
+  ~width: float=1.0,
```

Comment:
This will be a little simpler if it's an integer here for the `width`. Unless you have in mind a case for generator writers to need non integer values for line widths?